javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -388,6 +418,30 @@ export async function generateNetworkSpec(
         }
       }
 
+      for (const cmd of parachain.chain_spec_modifier_commands || []) {
+        const cmdHasRawSpec = cmd.some(arg => RAW_CHAIN_SPEC_IN_CMD_PATTERN.test(arg));
+        const cmdHasPlainSpec = cmd.some(arg => PLAIN_CHAIN_SPEC_IN_CMD_PATTERN.test(arg));
+    
+        if (cmdHasRawSpec && cmdHasPlainSpec) {
+          console.error(
+            decorators.red(
+              `Chain spec modifier command references both raw and plain chain specs!\n\t${cmd}`,
+            ),
+          );
+          process.exit();
+        }
+        else if (cmdHasRawSpec) parachainSetup.rawChainSpecModifierCommands.push(cmd);
+        else if (cmdHasPlainSpec) parachainSetup.chainSpecModifierCommands.push(cmd);
+        else {
+          console.error(
+            decorators.red(
+              `Chain spec modifier command does not attempt to reference a chain spec path!\n\t${cmd}`,
+            ),
+          );
+          process.exit();
+        }
+      }
```

Comment:
I think we should not mix single line if/else with brackets one. If there is only a single function call, single line is of for 1 if but when you have multiple, brackets are better. @wirednkod @pepoviola What do you think ?

```suggestion
        if (cmdHasRawSpec && cmdHasPlainSpec) {
          console.error(
            decorators.red(
              `Chain spec modifier command references both raw and plain chain specs!\n\t${cmd}`,
            ),
          );
          process.exit();
        }
        else if (cmdHasRawSpec) {
          parachainSetup.rawChainSpecModifierCommands.push(cmd);
        else if (cmdHasPlainSpec) {
          parachainSetup.chainSpecModifierCommands.push(cmd);
        } else {
          console.error(
            decorators.red(
              `Chain spec modifier command does not attempt to reference a chain spec path!\n\t${cmd}`,
            ),
          );
          process.exit();
        }
      }
```

Reply:
I agree - makes code more readable

Reply:
Agree 👍